risk-map/yaml/risks.yaml

```
@@ -554,6 +717,25 @@ risks:
           Mitigation involves robust filtering and processing of inputs and outputs.
           Additionally, thorough training, tuning, and evaluation processes help fortify
           the model against prompt injection attacks.
+    mappings:
+      mitre-atlas:
+        - AML.T0051
+      stride:
+        - tampering
+        - elevation-of-privilege
+      owasp-top10-llm:
+        - LLM01
+    lifecycleStage:
+      - evaluation
+      - runtime
```

Should only be runtime, no? 


```suggestion
    lifecycleStage:
      - runtime
```